src/include/common/types/types.h

```diff
@@ -87,6 +90,7 @@ class DataType {
 public:
     DataTypeID typeID;
     std::unique_ptr<DataType> childType;
+    uint64_t numElementsInTensor = UINT64_MAX;
```

Comment:
numElementsInTensor -> fixedNumChildren; or fixedNumElementsInList; if you want to be more verbose.

Reply:
RENAMED